app/src/main/java/com/epfl/beatlink/viewmodel/map/user/MapUsersViewModel.kt

```diff
@@ -94,13 +94,23 @@ open class MapUsersViewModel(private val repository: MapUserRepository) : ViewMo
   }
 
   /** Delete a user from the database. */
-  fun deleteMapUser() {
-    viewModelScope.launch {
+  fun deleteMapUser(): Boolean {
```

Comment:
Needs to treat the case where the data does not exist. Needs to not give a failure on non existence of the data.